internal/buildscripts/packaging/choco/splunk-otel-collector/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,156 @@
+﻿$ErrorActionPreference = 'Stop'; # stop on all errors
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+. $toolsDir\common.ps1
+
+
+$installation_path = "$drive" + "\Program Files\Splunk\OpenTelemetry Collector"
+$program_data_path = "$drive" + "\ProgramData\Splunk\OpenTelemetry Collector"
+$config_path = "$program_data_path\agent_config.yaml"
+
+echo "Checking configuration parameters ..."
+$pp = Get-PackageParameters
+
+$SPLUNK_ACCESS_TOKEN = $pp['SPLUNK_ACCESS_TOKEN']
+$SPLUNK_INGEST_URL = $pp['SPLUNK_INGEST_URL']
+$SPLUNK_API_URL = $pp['SPLUNK_API_URL']
+$install_dir = $pp['install_dir']
+$SPLUNK_HEC_TOKEN = $pp['SPLUNK_HEC_TOKEN']
+$SPLUNK_HEC_URL = $pp['SPLUNK_HEC_URL']
+$SPLUNK_TRACE_URL = $pp['SPLUNK_TRACE_URL']
+$SPLUNK_BUNDLE_DIR = $pp['SPLUNK_BUNDLE_DIR']
+$SPLUNK_REALM = $pp['SPLUNK_REALM']
+
+
+# create config files
+create_program_data
+
+# get param values from config files if they exist
+if (!$SPLUNK_ACCESS_TOKEN) {
+    $SPLUNK_ACCESS_TOKEN = get_value_from_file -path "$program_data_path\token"

```

Comment:
I don't believe this approach from the Smart Agent applies to the collector.

Reply:
Removing this code and now value can be passed through parameters while installing with choco or it will take from system environment registry. if in the system environment registry it is not specified then the script will set the default `1234` temporary value.